src/kixi/hecuba/weather/service.clj

```diff
@@ -1,7 +1,22 @@
 (ns kixi.hecuba.weather.service
+  (:require [clojure.java.io :as io]
+            [clojure.tools.cli :refer [parse-opts]])
   (:gen-class))
 
-(defn -main
-  "I don't do a whole lot ... yet."
-  [& args]
-  (println "Hello, World!"))
+(def cli-options
+  [["-c" "--config CONFIG" "Path to config edn file."
+    :default "config/settings.edn"]])
+
+(defn load-config [filename]
+  (with-open [r (io/reader filename)]
+    (read (java.io.PushbackReader. r))))
```

Comment:
the benefit of doing the `reader` rather than the `slurp` is that you can report the line number in the source edn file if there's an error. If you do `slurp` then you need to use `read-string` rather than `read`